redesign/case-studies/science-museum-group/online-collections.html

```diff
@@ -10,7 +10,7 @@
     <!-- End Google Tag Manager -->
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta name="description" content="dwyl is a worldwide community of people using technology to solve real problems.">
+    <meta name="description" content="Find out more about the Online Databank we made for the Science Museum Group, so that all of the SMG's data and artefacts could be accessible to the public">
```

Comment:
Find out more about the Online Databank app we built for the Science Museum Group - a public and accessible register for all of their artefacts and associated data

Reply:
Online Collections (it's not an 'app' per se)